src/wp-admin/includes/widgets.php

```diff
@@ -129,7 +129,7 @@ function wp_list_widget_controls( $sidebar, $sidebar_name = '' ) {
  * @global array $wp_registered_widgets
  *
  * @param array $params
- * @return array
+ * @return array Modified params array.
```

Comment:
I'll change this to "Widget control arguments." as the function's description is "Retrieves the widget control arguments."